mws/mws.py

```diff
@@ -691,6 +691,20 @@ def get_my_price_for_asin(self, marketplaceid, asins, condition=None):
                     ItemCondition=condition)
         data.update(utils.enumerate_param('ASINList.ASIN.', asins))
         return self.make_request(data)
+    
+    def get_my_fees_estimate(self, marketplaceid, asin, price, is_fba=False):
+        data = dict(Action='GetMyFeesEstimate')
+        data.update({
+            'FeesEstimateRequestList.FeesEstimateRequest.1.MarketplaceId': marketplaceid,
+            'FeesEstimateRequestList.FeesEstimateRequest.1.IdType':'ASIN',
+            'FeesEstimateRequestList.FeesEstimateRequest.1.IdValue':asin,
+            'FeesEstimateRequestList.FeesEstimateRequest.1.IsAmazonFulfilled':'true' if is_fba else 'false',
+            'FeesEstimateRequestList.FeesEstimateRequest.1.Identifier':'42',
+            'FeesEstimateRequestList.FeesEstimateRequest.1.PriceToEstimateFees.ListingPrice.Amount':str(price),
+            'FeesEstimateRequestList.FeesEstimateRequest.1.PriceToEstimateFees.ListingPrice.CurrencyCode':'USD'
+        })
+        return self.make_request(data)
+
 
```

Comment:
This version of the method is too specific to a certain use-case. It will need to be made more generic for all use-cases.

Some particulars:

* `IdType` can be either to "ASIN" or "SellerSKU". It should be passed as a possible argument.
* Argument `asin` is specific, and should be made more generic.
* `Identifier` should be changeable, so that users can write their own.
  * We can add an option to fill in unique values on-the-fly if necessary.
* `PriceToEstimateFees.ListingPrice` should be able to take any `CurrencyCode` to account for international users.
* The method, in general, can take multilple FeeEstimateRequests per request, as a batch. The method should be refactored to account for users who wish to send batch requests.

You can leverage the newer `utils.enumerate_keyed_param` method to construct the parameter set:

https://github.com/python-amazon-mws/python-amazon-mws/blob/a46eb438182546741137e1780048be538e6d98b6/mws/utils.py#L155-L203